packages/swap/prisma/schema.prisma

```diff
@@ -52,6 +52,7 @@ model SwapDepositChannel {
   swaps                 Swap[]
   failedSwaps           FailedSwap[]
   estimatedExpiryAt     DateTime?
+  openedWithCfBroker    Boolean      @default(false)
```

Comment:
its not the chainflip broker if somebody else is running the backend.. so maybe just `openedWithBroker`/`openedThroughBackend`